src/packageAPI.ts

```diff
@@ -170,12 +171,12 @@ export async function queryDependencyRubyGems(dependency: string, rateLimiter: P
 	const response = await (await fetch("https://rubygems.org/api/v1/versions/" + dependency + "/latest.json")).json()
 	const data = response as { version: string }
 
-	return { name: dependency, data: { version: data.version } }
+	return { name: dependency, data: { version: data.version, link: "https://rubygems.org/gems/" + dependency } }
```

Comment:
How difficult is it to support user defined npm/pypi/ruby links in a configuration file?
It would help if an organisation is using its own npm/pypi/ruby repository.

Reply:
Simple